conversations/document_based.py

```diff
@@ -3,11 +3,10 @@
 from langchain.memory import VectorStoreRetrieverMemory
 from langchain.text_splitter import CharacterTextSplitter
 from langchain.chains import ConversationChain
-from langchain.prompts import PromptTemplate
-from langchain.agents import Tool, initialize_agent, AgentType, load_tools
```

Comment:
let's not get rid of the agent. In the near-term, we will instead want to have a UI toggle to enable it instead.